packages/signature/src/lib/__test__/unit/util.spec.ts

```diff
@@ -170,6 +208,23 @@ describe('publicKeyToJwk', () => {
     const jwk2 = publicKeyToJwk(p256HexPublicKey, Alg.ES256, 'myKey2')
     expect(jwk2.kid).toBe('myKey2')
   })
+
+  it('converts a valid EDDSA hex public key to JWK', async () => {
+    const jwk = publicKeyToJwk(eddsaHexPublicKey, Alg.EDDSA)
+    const { d: _d, ...eddsaPublicKey } = eddsaKey
+    expect(jwk).toEqual(eddsaPublicKey)
+  })
+
+  it('works', async () => {
```

Comment:
Best test description that I've read in a long time 🤣  "well.. it. just works!"

```suggestion
  it('ensures hex consistency across standard and polyfill implementations', async () => {
```

Reply:
Lol, forgot this one